spec/features/manage_cmr/subscriptions/create_subscriptions_spec.rb

```diff
@@ -52,7 +49,6 @@
             # This is necessary to load the delete button for the subscription
             # clean up.
             # TODO: Remove when we can reset_provider
-            allow_any_instance_of(SubscriptionPolicy).to receive(:destroy?).and_return(true)
```

Comment:
Can remove the top comment here, the TODO should be updated with the CMR ticket number (6332)